src/pudl/metadata/resources/nrelatb.py

```diff
@@ -93,7 +99,10 @@
         "field_namespace": "nrelatb",
     },
     "core_nrelatb__yearly_technology_status": {
-        "description": "Annual table indicating whether technologies in the ATB scenarios are mature or default.",
+        "description": (
+            "Annual table indicating whether technologies in the ATB scenarios are mature,"
+            "and whether they are the default technologies."
```

Comment:
Do you know what mature/default mean in this context?

Reply:
Maturity is described here: https://atb.nrel.gov/electricity/2024/definitions
Default just means that they are the default values/settings, I believe.

Reply:
Would be good to include those in the description I think!

Reply:
Added this to maturity, I'm leaving default as is for now.